Front-End/challenge2/script.js

```diff
@@ -5,3 +5,171 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+
+let a;
+let b;
+let operation;
+let total = 0;
+let prev = false;
+let fromEqual = false;
+
+const display = document.getElementById("display");
+const zero = document.getElementById("zero");
+const one = document.getElementById("one");
+const two = document.getElementById("two");
+const three = document.getElementById("three");
+const four = document.getElementById("four");
+const five = document.getElementById("five");
+const six = document.getElementById("six");
+const seven = document.getElementById("seven");
+const eight = document.getElementById("eight");
+const nine = document.getElementById("nine");
+const add = document.getElementById("add");
+const subtrack = document.getElementById("subtrack");
+const multiplication = document.getElementById("multiplication");
+const division = document.getElementById("division");
+const equals = document.getElementById("equals");
+
+const clean = () => {
+  display.textContent = '';
+}
+
+const result = () => {
+  if(!display.textContent) return;
+  if(fromEqual) {
```

Comment:
for readability, add a blank line after each `if` statement and before each `return` we can solve some of this issues with prettier and eslint rules